include/PositionLine.h

```diff
@@ -38,19 +38,30 @@ class PositionLine : public QWidget
 	Q_OBJECT
 	Q_PROPERTY(bool tailGradient MEMBER m_hasTailGradient)
 	Q_PROPERTY(QColor lineColor MEMBER m_lineColor)
+	Q_PROPERTY(QColor recordingColor MEMBER m_recordingColor)
 public:
 	PositionLine(QWidget* parent, Song::PlayMode playMode);
 
+	bool isRecording() const {
+		return m_isRecording;
+	}
+
+	void setRecording(bool recording) {
+		m_isRecording = recording;
+	}
+
```

Comment:
```suggestion
	bool isRecording() const { return m_isRecording; }

	void setRecording(bool recording)
	{
		if (m_isRecording != recording)
		{
			m_isRecording = recording;
			update();
		}
	}

```

This would remove the need to manually call `update()` in calling code following each call to `setRecording()`. You may want to move this implementation to `PositionLine.cpp` though.